project/src/com/google/daggerqueryui/scripts/binding_graph.js

```diff
@@ -138,6 +167,50 @@ const bindingGraph = (function() {
     }).length !== 0;
   }
 
+  /**
+   * Supports click and double click events in the network object.
+   * @param {vis.Network} network
+   */
+  function supportEventsRecognition(network) {
+    // An event for managing children nodes.
+    network.on("doubleClick", function (params) {
+      // Checks if any node was selected.
+      if (params.nodes.length === 0) {
+        return;
+      }
+
+      const nodeId = params.nodes[0];
+      const nodeTitle = nodes.get(nodeId).title;
+
+      // If the dependencies are not currently shown, we prefer to draw them all.
+      // Otherwise, we hide all children, even if they are not all were presented.
+      if (getAllEdgesToDeps(nodeId).length === 0) {
+        queryExecutor.processQuery(['deps', nodeTitle], false);
+      } else {
+        bindingGraph.deleteDeps(nodeTitle);
+      }
+    });
+
+    // An event for managing parent nodes.
+    network.on("oncontext", function (params) {
```

Comment:
It looks like from the vis documentation this is the event that is triggered when the user clicks with the right mouse button. https://visjs.github.io/vis-network/docs/network/

Will the chrome context menu also pop up when the user right clicks?

Reply:
I see you've fixed this in the following PR :)
Resolving.